CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# CONTRIBUTING
+
+Contributions to the XRPL-Standards repo are free, open source, and follow the following document publishing process.
+
+Any XRPL-Standards document can be referred to interchangeably as an "XLS" (XRPL Standard), "RFC" (Request for Comments), or "document".
+
+## Summary
+
+Copyright on all content is subject to the terms of the [MIT License](LICENSE).
+
+All contributors grant everyone:
+
+Copyright – a royalty-free license to use the copyrights for their contributions.
+Patent – a commitment to license on a royalty-free basis their essential patent claims reading on their contributions.
+
+## Background
+
+The work of the XRP Ledger community is open source, collaborative, and welcoming of all contributors participating in good faith. [Learn more about the XRP Ledger at XRPL.org](https://xrpl.org/).
+
+This XRPL-Standards process is influenced by the well-established [IETF RFC process](https://www.ietf.org/standards/process/informal/).
+
+## Process
+
+The XRPL-Standards process attempts to be easy to use, but also rigorous enough that there are permalinks to revisions of documents for reference purposes.
+
+### Gathering Feedback Before Submitting
+
+Please gather community input before opening a PR. Collecting such feedback helps to refine your concept. This step is required.
+
+Start a [Discussion](https://github.com/XRPLF/XRPL-Standards/discussions) under this repo.
+
+Discussions are suitable for early work-in-progress: ask, suggest, add, and make sweeping changes.
+
+Next, the discussion should be converted to a **PULL REQUEST** which adds the standard, as a Markdown file, to the repo with one or more commits. At this time, the discussion should be archived and closed from further comments, with any further comments being posted on the PR for fine-tuning and alignment with implementation (as appropriate).
```

Comment:
I wonder about making L34 more specific to describe when a discussion moves to a PR. Maybe we also want authors to provide warning before closing a discussion for comment.

"When a discussion has produced a well refined standard, the standard author should work with the repository maintainers to move from discussion to **PULL REQUEST**. Authors should post a comment to a discussion warning that it will be closed multiple days before closing the discussion for further comment, thus allowing those engaged with the discussion time to submit final commentary.

When moving to a pull request, add the standard, as a Markdown file, to the repo..."